src/bug-filing/azure-boards/create-azure-boards-issue-filing-url.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { title } from '../../content/strings/application';
+import { getIssueDetailsHtml } from '../common/get-issue-details-html';
+import { HTTPQueryBuilder } from '../common/http-query-builder';
+import { IssueDetailsGetter } from '../common/issue-details-getter';
+import { IssueFilingUrlStringUtils, IssueUrlCreationUtils } from '../common/issue-filing-url-string-utils';
+import { AzureBoardsBugFilingSettings } from './azure-boards-bug-filing-service';
+
+const buildTags = (createBugData: CreateIssueDetailsTextData, standardTags: string[]): string => {
+    const tags = ['Accessibility', title, `rule:${createBugData.ruleResult.ruleId}`, ...standardTags];
+    return tags.join('; ');
+};
+
+export const createAzureBoardsIssueFilingUrlProvider = (
+    stringUtils: IssueUrlCreationUtils,
+    issueDetailsGetter: IssueDetailsGetter,
+    queryBuilderProvider: () => HTTPQueryBuilder,
+) => {
+    return (settingsData: AzureBoardsBugFilingSettings, bugData: CreateIssueDetailsTextData, environmentInfo: EnvironmentInfo) => {
+        const standardTags = stringUtils.standardizeTags(bugData);
```

Comment:
There's an odd division of responsibility between `standardizeTags` and `buildTags` (which was present in the old code too, I see). 

- `IssueFilingUrlStringUtils.standardizeTags` gets the WCAG standard tags out of a `CreateIssueDetailsTextData` and capitalizes them
- `buildTags` collects the rule tag out of a `CreateIssueDetailsTextData`, the WCAG standard tags passed in, and some hardcoded tags. 

Might it be cleaner to split as: 
- `standardizeTags` to be something like `getTagsFromIssueDetails` and have it return all the tags that depend on data from `CreateIssueDetailsTextData`
- `buildTags` takes that list (and doesn't need to take a `CreateIssueDetailsTextData`), adds the hardcoded tags, and return the joined string?

Reply:
`standardizeTags` is also being used on the git hub issue title, which don't include the rule id.

I think this will be reworked as part of our fit and finish. There are some thing I need to discuss with PM's before finalize the issue details for azure boards as they are not the same as for git hub right now but I don't have specs said issue details.

Reply:
Oh, wait... `standardizeTags` is also used to generate part of the title. (It is a happy coincidence that we use the same delimeter between a list of tags to put on an issue, and the list of WCAG standards we put in the title.)